docs/design/autoware-architecture/planning/index.md

```diff
@@ -112,7 +112,7 @@ As mentioned in the goal session, this planning module is designed to be extensi
 | Adaptive Cruise Control                      | Plan velocity to follow the vehicle driving in front of the ego vehicle. <br> <br> Reference implementation is in [Obstacle Stop Planner](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_stop_planner/), [Obstacle Cruise Planner](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_cruise_planner/).                                                                                            | - objects information                                                       | ![adaptive-cruise](image/features-adaptive-cruise.drawio.svg)                   |
 | Decelerate for cut-in vehicles               | Plan velocity to avoid a risk for cutting-in vehicle to ego lane. <br> <br> Reference implementation is in [Obstacle Cruise Planner](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_cruise_planner/).                                                                                                                                                                                                                         | - objects information                                                       | ![cut-in](image/features-cut-in.drawio.svg)                                     |
-| Surround Check at starting                   | Plan velocity to prevent moving when an obstacle exists around the vehicle. <br> <br> Reference implementation is in [Surround Obstacle Checker](https://autowarefoundation.github.io/autoware.universe/main/planning/surround_obstacle_checker/).                                                                                                                                                                                                           | - objects information                                                       | ![surround-check](image/features-surround-check.drawio.svg)                     |
+| Surround Check at starting                   | Plan velocity to prevent moving when an obstacle exists around the vehicle. <br> <br> Reference implementation is in [Surround Obstacle Checker](https://autowarefoundation.github.io/autoware.universe/main/planning/surround_obstacle_checker/). Enable flag in parameter: `use_surround_obstacle_check true` in 'tier4_planning_component.launch.xml'                                                                                                     | - objects information                                                       | ![surround-check](image/features-surround-check.drawio.svg)                     |
```

Comment:
[imo] It is better to paste link to the flag in `tier4_planning_component.launch.xml`.

FYI: [imo] means "in my opinion".